corebuild/integration/ILLink.Tasks/ILLink.Tasks.targets

```diff
@@ -109,7 +109,7 @@
   <UsingTask TaskName="CompareAssemblySizes" AssemblyFile="$(LinkTaskDllPath)" />
   <Target Name="_CompareLinkedAssemblySizes"
           AfterTargets="ComputeLinkedFilesToPublish"
-          DependsOnTargets="_ComputeManagedAssembliesToLink;_ComputeLinkedAssemblies"
+          DependsOnTargets="_ComputeManagedAssembliesToLink;_ComputeLinkedAssemblies;_FindNativeDeps"
```

Comment:
Isn't it enough to specify just _FindNativeDeps here? It depends on _ComputeLinkedAssemblies, which depends on ILLink, which depends on _ComputeManagedAssembliesToLink.

Reply:
That would have the same behavior, though I think it doesn't hurt to be explicit in case `_FindNativeDeps` changes. I've been trying to add dependencies on exactly the targets that produce the itemgroups that a given target uses. In this case, `_FindNativeDeps` was left over because I had been considering also printing a comparison of the removed native assemblies, so it should actually be removed here. Thanks!